recipes/pynhd/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pynhd" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1c35dd89301eeb6ba4683b372fa811ed9ad135e0b621f9610bd9dff24dd0e255
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
+  run:
+    - python >=3.6
+    - setuptools
+    - pygeoutils
+    - pygeoogc
+    - networkx
+
+test:
+  imports:
+    - pynhd
```

Can we try to add ``pip check`` here please?
```suggestion
    - pynhd
  requires:
    - pip
  commands:
    - pip check
```